README.md

```diff
@@ -243,7 +243,7 @@ is described in 1 step and a step later mentions the same filename, they both re
    ```
    __IMPORTANT:__ If memory causes issues, `--buffer_size 500` (or something similar) can be used 
    to reduce memory usage (at the cost of speed).
-8. Lift over not-annotated VCFs (output from `train-data-creator`) to GRCh38 using `liftover_variants.sh`:
+8. Lift over not-annotated VCFs (output from `train-data-creator`) to GRCh38 using `liftover_variants.sh` (and the Picard 3.0.0 image, available [here](https://download.molgeniscloud.org/downloads/vip/images/)):
```

Comment:
Adjust link to go directly to https://download.molgeniscloud.org/downloads/vip/images/picard-3.0.0.sif or make documentation more generic to refer to "latest Picard image available here"?

Reply:
Disagree. That would require constant monitoring of the picard image on already expensive maintaining of documentation.